<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qolo_data</class>
 <widget class="QWidget" name="qolo_data">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>qolo_data</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QPushButton" name="open">
       <property name="text">
        <string>打开</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="save">
       <property name="text">
        <string>保存</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="update">
       <property name="text">
        <string>更新</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="all_label">
       <property name="text">
        <string>标签</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="del">
       <property name="text">
        <string>删除</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="up">
       <property name="text">
        <string>上一个</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="down">
       <property name="text">
        <string>下一个</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="g_layout">
     <property name="spacing">
      <number>1</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="list_layout">
       <property name="spacing">
        <number>2</number>
       </property>
       <item>
        <widget class="QListWidget" name="curr_label">
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>360</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="file_list">
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>360</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
